### PR TITLE
4 lines of code; please review :-)

### PR DESCRIPTION
**Pull request by songgao**
This PR changes chat CLI to show an error message if user tries to specify topic name for CHAT

r? `any(`@patrickxb @mmaxim @maxtaco`)` Thanks!

### CODE REVIEW

File: go/client/chat_cli_utils.go [hunk 1]
```diff
@@ -125,6 +126,9 @@ func parseConversationResolver(ctx *cli.Context, tlfName string) (resolver conve
 	if resolver.TopicType, err = parseConversationTopicType(ctx); err != nil {
 		return resolver, err
 	}
+	if resolver.TopicType == chat1.TopicType_CHAT && len(resolver.TopicName) != 0 {
+		return resolver, errors.New("multiple topic names are not yet supportted with chat conversations")
```
Comment (maxtaco) [inline go/client/chat_cli_utils.go]: error message is misspelled. Just say `"multiple topic names are not yet supported"`

How does one get into this situation? Will the conversation be broken in this case?  Have you seen i t?

Comment (songgao) [inline go/client/chat_cli_utils.go]: `keybase chat send --topic-name test hi` tries to insert a message to set topic name which seems broken for now (but worked at some point I think). `keybase chat read --topic-name test` would simply resolve to the default conversation without any topic name, which is not broken but also can be a little be confusing. Doing this before hitting server gives user clear message (without RTT delays) that this won't work, which is kind of nice :-)

I'll fix the error message; Thanks!

File: go/client/chat_cli_utils.go [hunk 2]
```diff
@@ -127,7 +127,7 @@ func parseConversationResolver(ctx *cli.Context, tlfName string) (resolver conve
 		return resolver, err
 	}
 	if resolver.TopicType == chat1.TopicType_CHAT && len(resolver.TopicName) != 0 {
-		return resolver, errors.New("multiple topic names are not yet supportted with chat conversations")
+		return resolver, errors.New("multiple topic names are not yet supported")
```
Comment (maxtaco) [inline go/client/chat_cli_utils.go]: Sorry, I'm still not totally clear on this.  Do we mean to say `multiple topics are not yet supported`?

Comment (songgao) [inline go/client/chat_cli_utils.go]: hmm.. I thought it was the same but I can remove the "names" as well. Is this what you meant or did I miss something else?

**Comment by songgao** [comment]
Also I noticed with the newest gregor, multiple topics for dev conversations is broken as well. I'll look into it in the same ticket (as it's probably a bug in service)!

File: go/client/chat_cli_utils.go (hunk 2 above)
Comment (maxtaco) [inline go/client/chat_cli_utils.go]: Yeah, that's what I meant.  I just thought that error was still confusing.  Does this new text match the error case you have in mind?

Comment (songgao) [inline go/client/chat_cli_utils.go]: Yup it does. Well what I had in mind is dev conversations can still have multiple topic per TLF but I guess for CLI interface, not mentioning dev conversations is fine!

**Comment by maxtaco** [comment]
Ok, 👍 after the change to the most recent iteration of the error message

**Comment by songgao** [comment]
Cool; Thanks Max!

**Comment by songgao** [comment]
I can't reproduce the NotInConversation error any more and multiple topics in dev conversation is magically working again ... Merging and closing this ticket for now!

**Pull request merged**